test/integration/an_existing_user_can_check_out_to_place_an_order_test.rb

```diff
@@ -38,4 +38,17 @@ class AnExistingUserCanCheckOutToPlaceAnOrderTest < ActionDispatch::IntegrationT
       assert page.has_content?('0')
     end
   end
+
+  test "an_existing_user_cant_check_out_with_0_items" do
+    create_user
+    login_a_user
+
+    visit '/cart'
+    click_link 'Check Out'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.